assets/js/controllers/send.controller.js

```diff
@@ -35,37 +35,40 @@ function SendCtrl ($scope, AngularHelper, $log, Wallet, Alerts, currency, $uibMo
   $scope.fiatCurrency = Wallet.settings.currency;
   $scope.btcCurrency = Wallet.settings.btcCurrency;
   $scope.isBitCurrency = currency.isBitCurrency;
+  $scope.fromSatoshi = currency.convertFromSatoshi;
+  $scope.format = currency.formatCurrencyForView;
   $scope.isValidPrivateKey = Wallet.isValidPrivateKey;
-
-  $scope.defaultBlockInclusion = 1;
+  $scope.satoshi = currency.bitCurrencies.filter((c) => c.code === 'satoshi')[0];
 
   $scope.transactionTemplate = {
-    from: null,
-    destinations: [null],
-    amounts: [null],
     fee: 0,
+    size: 0,
     note: '',
-    maxAvailable: null,
+    from: null,
     surge: false,
-    blockIdx: null,
-    feeBounds: [0, 0, 0, 0, 0, 0],
-    sweepFees: [0, 0, 0, 0, 0, 0],
-    size: 0
+    amount: null,
+    maxAvailable: null,
+    maxSpendableAmount: null,
+    destinations: [null],
```

Comment:
This can be changed from a single value into an array (like you did with `amounts`).